librecad/src/main/console_dxf2png.cpp

```diff
@@ -79,7 +79,7 @@ bool slotFileExport(RS_Graphic* graphic,
 /// \param argv
 /// \return
 ///
-int console_dxf2png(int argc, char* argv[])
+int console_dxf2png(int argc, char** argv)
```

Comment:
`char** argv` is equivalent to `char* argv[]` as a function parameter

Reply:
(Msvc main signature)[https://learn.microsoft.com/en-us/cpp/cpp/main-function-command-line-args?view=msvc-170]

This is puzzling. `int foo(char **)` is the same as `int foo(char** const)`.

Again, could you share the exact linker error messages.

I may still have msvc on some old laptops. I can give it a try also.

Reply:
This is the error I was getting before this change:
```
main.obj : error LNK2019: unresolved external symbol "int __cdecl console_dxf2png(int,char * *)" (?console_dxf2png@@YAHHPEAPEAD@Z) referenced in function main
  Hint on symbols that are defined and could potentially match:
    "int __cdecl console_dxf2png(int,char * * const)" (?console_dxf2png@@YAHHQEAPEAD@Z)
..\..\windows\LibreCAD.exe : fatal error LNK1120: 1 unresolved externals
```
And in any case, I believe the function declaration should match the function definition.

Weirdly enough, `console_dxf2pdf` seems to be the same, the definition doesn't match the declaration, but in this case the compiler / linker do not mind. I have no idea what's going on here, but since fixing the mismatch has fixed the error, I just went with it.